experiments/opencog/pattern_matcher_vqa/run.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+python3 pattern_matcher_vqa.py \
+    --questions filtered_questions.txt \
+    --model-kind SPLITMULTIDNN \
+    --atomspace atomspace_val.scm \
+    --multidnn-model /home/noskill/projects/models/multi/visual_genome \
```

Comment:
I would commit paths to ```/mnt/fileserver``` instead. In such case at least in our team anyone will be able to run it without changing ```run.sh```